cli/command/stack/remove.go

```diff
@@ -13,56 +14,63 @@ import (
 )
 
 type removeOptions struct {
-	namespace string
+	namespaces []string
 }
 
-func newRemoveCommand(dockerCli *command.DockerCli) *cobra.Command {
+func newRemoveCommand(dockerCli command.Cli) *cobra.Command {
 	var opts removeOptions
 
 	cmd := &cobra.Command{
 		Use:     "rm STACK",
 		Aliases: []string{"remove", "down"},
 		Short:   "Remove the stack",
```

Comment:
the CLI reference documentation on docs.docker.com is generated from this output, in combination with the markdown in this repo (https://github.com/docker/docker/pull/32110#pullrequestreview-30006189), so perhaps no changes are needed, but you could check if there's other markdown files (here, and in the docs repo) that include the USAGE output